bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
My fault not to look at that before, but the folder structure is not appropriate.

 * `bootstrap.sh` is not a dev utility script (as `gradlew` is), so it should not lie in the root folder.
 * `bootstrap` directory is not part of the main artifact (the plugin itself) sources, so it should not lie in the `src/main` folder

What I suggest:

```
.
├── src/
│   ├── main/
│   └── test/
├── samples/
│   └── bootstrap/ ### here lie all the bootstrap sample/
│       ├── boostrap.sh
│       └── template/
│           ├── gradle/
│           │   └── ...
│           ├── build.gradle
│           ├── gradlew
│           └── gradlew.bat
├── gradle/
│   └── ...
├── build.gradle
├── .gitignore
├── gradlew
├── gradlew.bat
├── LICENSE.txt
└── README.adoc
```

or, if you prefer to split and lie under src:

```
.
├── src/
│   ├── bootstrap/
│   │   ├── script/
│   │   │   └── bootstrap.sh
│   │   └── template/
│   │       ├── gradle/
│   │       │   └── ...
│   │       ├── src/
│   │       │   └── gatling/
│   │       │       ├── scala/
│   │       │       │   └── ...
│   │       │       └── resources/
│   │       │           └── ...
│   │       ├── build.gradle
│   │       ├── gradlew
│   │       └── gradlew.bat
│   ├── main/
│   │   └── ...
│   └── test/
│       └── ...
```

Should we provide a README with basic introduction to the sample project?